api/app/Models/User.php

```diff
@@ -447,6 +447,20 @@ public function filterByEquity(Builder $query, array $equity): Builder
         return $query;
     }
 
+    public function filterByPoolCandidateStatus(Builder $query, ?string $poolCandidateStatus): Builder
+    {
+        if(empty($poolCandidateStatus)){
+            return $query;
+        }
+
+        if($poolCandidateStatus == 'AVAILABLE'){
+            $query->where('pool_candidate_status', 'AVAILABLE');
```

Comment:
I don't think the user has a pool_candidate_status. This filter will actually be a bit more complicated on the User model, since it only applies to the Pools we're filtering by.